src/bacnet_server/resources/point/point_name.py

```diff
@@ -0,0 +1,54 @@
+import copy
+
+from flask_restful import marshal_with, abort, reqparse
+
+from src.bacnet_server.bac_server import BACServer
+from src.bacnet_server.models.model_point import BACnetPointModel
+from src.bacnet_server.models.model_priority_array import PriorityArrayModel
+from src.bacnet_server.resources.mod_fields import point_fields
+from src.bacnet_server.resources.point.point_base import BACnetPointBase
+
+
+class BACnetPointName(BACnetPointBase):
+    parser_patch = reqparse.RequestParser()
+    parser_patch.add_argument('object_name', type=str, required=False)
```

Comment:
Remove this one, coz we are updating through `object_name` to others attributes and add:

```
parser_patch.add_argument('object_type', type=str, required=False)
parser_patch.add_argument('address', type=int, required=False)
```

We can say, replace with that one with this one.